docs/snap-confine.rst

```diff
@@ -126,6 +133,18 @@ FILES
 
 	Description of the seccomp profile.
 
+`/run/snapd/ns/.lock`:
+
+	Global lock acquired to initialize namespace sharing feature
```

Comment:
FYI, I found the .lock file under-documented. I'm not sure where it would be best to document it, but it wasn't part of the google doc spec and I had to put it together myself why it was needed.